src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -184,8 +179,11 @@ public ListBoxModel doFillConfigTypeItems() {
     private boolean ignoreUnstableBuilds;
     private boolean persistConstraintLog;
 
+    private String reportFiles;
+
     @DataBoundConstructor
-    public PerformancePublisher(int errorFailedThreshold,
+    public PerformancePublisher(String reportFiles,
```

Comment:
This will break exitsing pipeline script, right? Will it break GUI-based existing configs after upgrade?